### ISSUE TITLE
Binding C-0, C-1,...

### ISSUE THREAD

**Issue by mikegant** [tmux/tmux#1308]
See next comment.

**Comment by mikegant** [comment]
I am trying to bind C-0, C-1, et. al. to select a session using this key binding
bind C-0 attach-session -t =0
bind C-1 attach-session -t =1
... etc.

And it just does not work. I thought that I might have the syntax wrong so I changed the key to M-0, M-1,... and this works.

What is unique about using Control + a number for a keybinding? I've looked through the man page and can't find anything. There are other key bindings that use the Control key + a letter but with a number does not appear to work. I tried to bind another action other than 'attach-session' and it doesn't work so I think that Control + a number must be reserved or special in some way.
